### ISSUE TITLE
ASR Result on InjecAgent

### ISSUE THREAD

**Issue by Bluedask** [albert-y1n/PISmith#3]
Hello, thanks for your contribution. I follow your script to train and eval PISmith on InjecAgent (secalign-8b based). I get the RL training processs in the image below and only get 0.2/0.52 ASR of Pass@1 and Pass@10. I just want to know the reasons of this result and could you please provide the checkpoint attacking InjecAgent.

Thank you for help.

<img width="50%" height="685" alt="Image" src="https://github.com/user-attachments/assets/b25b22ca-90a9-4a19-a129-17fa509af6d2" />

**Comment by Bluedask** [comment]
Thank you very much for sharing the checkpoint! I evaluated it locally and can reproduce results close to the paper. 

I have one follow-up question: if we want to train a comparable checkpoint from scratch, are there any important details beyond the released script/config that we should pay attention to? (such as training steps, learning rate, or others)

Thanks again for your help!
